Updating the DevTools screenshots to match new Fluent icons (#1377)
MicrosoftDocs/edge-developer: microsoft-edge/devtools-guide-chromium/index.md

> title: Microsoft Edge (Chromium) Developer Tools overview  

Reviewer: ```suggestion
title: Microsoft Edge Developer Tools overview  
```
AND change # h1 top-of-page title to match.  
AND in first sentence of body, maybe change:
When you install Microsoft Edge
to:
When you install Microsoft Edge (Chromium)

Author: Done

> title: Microsoft Edge (Chromium) Developer Tools overview  

Reviewer: Change # h1 top-of-page title from:
Microsoft Edge (Chromium) Developer Tools overview
to:
Microsoft Edge Developer Tools overview

AND in first sentence of body, change:
When you install Microsoft Edge
to:
When you install Microsoft Edge (Chromium)

author: MSEdgeTeam  
ms.author: msedgedevrel  
ms.date: 08/06/2021  
ms.topic: conceptual  
ms.prod: microsoft-edge  

> ms.technology: devtools  

Reviewer: Change first sentence tagging from
now referred to as `DevTools` .
to
now referred to as `DevTools` .
(delete space before period)

To open and explore the DevTools, use one any of the following actions.  

*   Hover on any element on the webpage, open the contextual menu \(right-click\), and then select **Inspect**.  This action opens the **Elements** tool.  
*   Select `F12`.  
*   Select `Ctrl`+`Shift`+`I` on Windows/Linux or `Command`+`Option`+`I` on macOS.  
    
:::row:::  

>    :::column span="":::  

Reviewer: Likely out of scope for this PR: Remove 2-column layout for images?  Ask mgmt, who mentioned moving away from 2-col.  eg when I'm reviewing the rendered articles, the 2-column images are too small to review, so I have to go out of my way to magnify (click) each one, then I see the big image w/ no article context.

> When you open the DevTools, it docks to the left of your browser.  To change the docked location of the DevTools, complete the following actions.  

Reviewer: In a list item, add a period after "Use the mouse"

    
:::row:::  
   :::column span="":::  

>       :::image type="complex" source="./media/devtools-intro-docking-left.msft.png" alt-text="Select Dock To Left" lightbox="./media/devtools-intro-docking-left.msft.png":::  

Reviewer: Global in this file and possibly others: lowercase UI commands to match latest UI; "Dock to left" at least at:  DevTools > ... > hover over the docking icons.   Also in Command Menu.
That's sort of related to this PR's purpose of updating screenshots to reflect latest UI.

       Screenshot of the DevTools display of this article in an emulated mobile phone  
    :::image-end:::  
    

> *   The Tools tab group is a group of tabs that represent different tools that are used in different scenarios.  You may customize each of the tools and each tool may change based on the context.  To open a dropdown menu of more tools, select the **More tabs** \(`>>`\) button.  Each of the tools is introduced later in the following section.  

Reviewer: comment: These are good corrections to conform to Microsoft Writing Style Guide for readability.  Even if main purpose of PR is images not text.

Reviewer: ```suggestion
*   The Tools tab group is a group of tabs that represent different tools that are used in different scenarios.  You can customize each of the tools, and each tool may change based on the context.  To show a tool that is on a tab that's hidden, select the **More tabs** \(`>>`\) button.   To add a tool to the main toolbar or to the drawer, select the **More tools** (`+`) button.  Each of the tools is introduced later in the following section.  
```
Not sure where + and >> are covered.  Easier to do edit pass locally in VS Code.

> *   Next to the Tools tab group are optional error and issues shortcuts.  The shortcuts display when JavaScript errors or issues occur on the current webpage.  The **Open Console to view # errors, # warnings** \(**JavaScript Errors**\) button displays a red circle with an `X` followed by the number of JavaScript errors.  To open the [Console][DevtoolsGuideConsoleIndex] and learn about the error, select the **JavaScript Errors** button.  The **Open Issues to view # issues** \(**Issues**\) button is a blue message icon followed by the number of issues.  To open the [Issues][DevtoolsGuideIssuesIndex] tool, select **Issues** button.  

Reviewer: ```suggestion
*   Next to the Tools tab group are optional error and issues shortcuts.  The shortcuts display when JavaScript errors or issues occur on the current webpage.  The **Open Console to view # errors, # warnings** \(**JavaScript Errors**\) button displays a red circle with an `X`, followed by the number of JavaScript errors.  To open the [Console][DevtoolsGuideConsoleIndex] and learn about the error, select the **JavaScript Errors** button.  The **Open Issues to view # issues** \(**Issues**\) button is a blue message icon followed by the number of issues.  To open the [Issues][DevtoolsGuideIssuesIndex] tool, select the **Issues** button.  
```

Reviewer 2: What if line 123 read:
The Tools tab group represent tools used in different scenarios. You may customize each of the tools and each tool may change based on the context. To open a dropdown menu of more tools, select the **More tabs** \(`>>`\) button. Each of the available tools is introduced later in the following section.

Reviewer: See the previous comment thread about line 123; suggest you delete or modify your comment and put it in that thread for line 123.

*   The **Settings** button displays a gear icon.  To open DevTools **Settings** webpage, select the **Settings** button.  The **Settings** webpage displays a menu to change **Preferences**, turn on **Experiments**, and much more.  

> *   The **Send Feedback** button displays torso with a chat bubble next to it.  To open the **Send Feedback** dialog, select the **Send Feedback** button.  The **Send Feedback** dialog allows you to enter information to describe what happened and automatically includes a screenshot.  Use it to connect with the DevTools team to report problems, issues, or suggest ideas.  

Reviewer: ```suggestion
*   The **Send Feedback** button displays a torso with a chat bubble next to it.  To open the **Send Feedback** dialog, select the **Send Feedback** button.  The **Send Feedback** dialog allows you to enter information to describe what happened and automatically includes a screenshot.  Use it to connect with the DevTools team to report problems, issues, or suggest ideas.  
```

> *   The **Customize and control Devtools** \(`...`\) button opens a dropdown menu.  It allows you to define where to dock the DevTools, search, open different tools, and much more.  

Reviewer: Global: change:
Devtools
to
DevTools

> The DevTools provides lots of features and functionality to use with your web product.  Access the different parts of the DevTools in many ways, but the fastest way to access the features you need is to use the command menu.  For more information, navigate to [Run commands with the Microsoft Edge DevTools Command menu][DevtoolsGuideCommandMenuIndex].  To open the command menu, complete one of the following actions.  

Reviewer: "complete one of the following actions" is robot words; unnatural.  
https://docs.microsoft.com/en-us/style-guide/top-10-tips-style-voice#write-like-you-speak

Author: I know you have a work item planned to address problems in this article. So I'll leave this as is for now and will only focus on changing screenshots from now on.

*   Select `Control`+`Shift`+`P` \(Windows, Linux\) or `Command`+`Shift`+`P` \(macOS\).  
*   Select **Customize And Control DevTools** \(`...`\), and then select **Run Command**.  

:::image type="complex" source="./media/devtools-intro-command-menu.msft.png" alt-text="Screenshot of the command menu in DevTools" lightbox="./media/devtools-intro-command-menu.msft.png":::  
Screenshot of the command menu in DevTools  
:::image-end:::  

> The command menu allows you to type commands to display, hide, or run features in the DevTools.  With the command menu open, enter the word **changes**, and then select **Show Changes**.  The **Changes** tool opens which is useful when you edit CSS, but is difficult to find in the DevTools UI.  

Reviewer: ```suggestion
The command menu allows you to type commands to display, hide, or run features in the DevTools.  With the command menu open, enter the word **changes**, and then select **Drawer: Show Changes**.  The **Changes** tool opens which is useful when you edit CSS, but is difficult to find in the DevTools UI.  
```
This is the convention I use.  It would be interesting to check repo convention, but regardless, I believe this (adding the command category eg "Drawer: " is most helpful to represent the UX.
